### DISCUSSION TITLE
Version 2.3.0

### DISCUSSION THREAD

**Discussion by guydavis** [guydavis/machinaris#989] [Announcements]
Version 2.3.0 of Machinaris is now available, for those that like to try the very latest Chia and Gigahorse releases.

### Added
 - Gigahorse: Optionally launch `chia-exporter` [metrics endpoint](https://github.com/Chia-Network/chia-exporter) for Prometheus reporting if env var `chia_exporter=true` is set.
### Updated
 - [Chia](https://github.com/Chia-Network/chia-blockchain/releases/tag/2.3.0) to v2.3.0 - misc improvements, see their release notes.
 - [Gigahorse](https://github.com/madMAx43v3r/chia-gigahorse/releases/tag/v2.3.0.giga36) to v2.3.0.giga36.
### Notes
 - Support for updated blockchain (and their bugs) DOES NOT imply my endorsement for them.  *Only run those you are comfortable with.*

<hr /><em>This discussion was created from the release <a href='https://github.com/guydavis/machinaris/releases/tag/2.3.0'>Version 2.3.0</a>.</em>

**Comment by danktankk** [comment]
Thank you for your excellent work on this project. 

I am having an issue getting staicoin to work in machinaris v2.3.0.  I waited about a week for the height of the blockchain to get right but after that happened, I am getting zero connectivity.  Its just plain offline.  Is Staicion down or dead perhaps?  The log files in chiadog are pretty sparse, so it seems more likely that this one went the way of the dinosaur.  Any help would be greatly appreciated.  Thank you again for your time and effort on this project!

**Reply by guydavis** [reply]
Sorry, don't know what to tell you about Staicoin.  It's always been flaky for me as well.  Currently, my local farmer is synced, but not common:
![image](https://github.com/guydavis/machinaris/assets/12936/9c3ffdfc-2ae9-4262-bf38-c58061aaf7a4)

I would recommend first trying to [refresh the Staicoin config](https://github.com/guydavis/machinaris/wiki/Troubleshooting#reset-blockchain-config), then [rebuild the entire Staicoin database]( https://github.com/guydavis/machinaris/wiki/Troubleshooting#reset-blockchain-database).

**Reply by danktankk** [reply]
thank you for the advice.  I will give it a shot.  Otherwise I will just delete it.  It hasnt connected once in almost a month now and just says worker not responding to pings.

**Reply by danktankk** [reply]
sorry for what may be a stupid question, but each of these forks needs its own DB, correct?  I read this "One workaround is to [download a Chia DB checkpoint manually](https://www.chia.net/downloads/)."

Can I use this instead of waiting for 5-6 days to get the database synced again?

**Comment by guydavis** [comment]
Yes, I don't know of a recent staicoin database download, so you'll have to
let it sync up from scratch over a few days. Try just the config file reset
first though.

**Reply by danktankk** [reply]
Thanks again,

I did try that and it did not help.  I guess ill have to try it the db resync.  It takes over 5 days now to do that :( If it fails a 2nd time, ill just accept defeat lol
